docs/usage/routing/parameters.rst

```diff
@@ -164,6 +164,28 @@ will be used for the value of the ``snake_case`` parameter.
 
 In this case, ``param`` is validated to be an *integer larger than 5*.
 
+Documentation for enum query parameters
+---------------------------------------
+
+Using :func:`~.params.Parameter` to modify the OpenAPI schema generated for query :term:`parameters <parameter>` that
+are enums can have surprising results. Since only one schema is generated per enum, calling :func:`~.params.Parameter`
```

Comment:
```suggestion
Since only one schema is generated per enum, calling :func:`~.params.Parameter`
```
And maybe instead mention how the docstring is used for the description by default?